pages/asks.tsx

```diff
@@ -19,9 +19,10 @@ export default function Asks() {
 
 	const askElements = useMemo(() => {
 		return Object.entries(content).reduce((acc, [id, ask]) => {
-			if (!showOnlyLikedAsks || (showOnlyLikedAsks && ask.isLiked)) {
+			if (!showOnlyLikedAsks || (showOnlyLikedAsks && ask["isLiked"])) {
```

Comment:
why `["isLiked"]` and not `.isLiked`?

ps. I see it repeats elsewhere... explain or refactor all

Reply:
noted